packages/hadron-document/src/element.ts

```diff
@@ -455,6 +466,14 @@ export class Element extends EventEmitter {
     }
   }
 
+  _isExpandable(): boolean {
+    return (
+      isValueExpandable(this.originalExpandableValue) ||
+      (this.isModified() &&
+        (this.currentType === 'Array' || this.currentType === 'Object'))
```

Comment:
I know that this is just preserving existing code, but the more I look at it, the less sense it makes, sorry 😅 If you edited the element to a non-expandable type, changed Object to String say, this would still return `true`, which just doesn't sound right? Feels like the check you are adding instead, the one that looks at the `currentType`, is actually the best way to check for this instead of checking the value and is the only thing we need to check for, does this makes sense or is there a case I'm missing where looking at the original value is required?

Reply:
Good callout! Removed everything except the current type check.